index.html

```diff
@@ -3,9 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
-    <title>Nep huiswerkopdracht</title>
+    <title>Nieuew titel</title>
 </head>
 <body>
     <h2>Dit is een andere titel, die gaat niet over Pim</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cum distinctio quod tenetur. Accusantium dignissimos expedita inventore minima necessitatibus omnis quae quibusdam voluptatum! Beatae eligendi, ex praesentium provident quod saepe voluptates.</p>
+
+    <article>
```

Comment:
Goed dat dit artikel erbij staat, alleen taalinstelling klopt niet helemaal. Staat nog in het spaans